docs/security-monitoring.md

```diff
@@ -23,11 +23,14 @@ Organizations using the agency add-on have the option to disable security monito
 
 ### Configuring Notifications
 Every user receives security notifications by email for all packages they have access to by default.
-Users can unsubscribe either from individual packages, or from all security notifications, if they not wish to receive any email notifications.
+Users can unsubscribe either from individual packages or from all security notifications if they do not wish to receive any email notifications.
 
-In addition to registered users receiving security notifications by email, you can configure notifications to be sent
-to other email addresses, e.g. a security mailinglist, or to a Slack channel.
-Create a notification channel for the desired destination and configure which notifications it should receive in the organization's security settings.
+The following notification channels are supported:
+* Email: send notifications to a configured email addresses list
```

Comment:
What is an "email addresses list", do you mean "list of email addresses" or "mailing list addresses"?

You should also start with an upper case after `:` if it is followed by a complete sentence. I'm also not sure on the choice of grammatical mood (imperative), you wrote "send notifications", is that an instruction? How about "Email: Sends notifications"?

For clearer layout you could make each notification type bold as well.